### PR TITLE
Add Ubuntu fallback to passphrase, add defaults

### PR DESCRIPTION
**Pull request by cbachert**
This PR adds fallback to passphrase in case fido2luks fails on Ubuntu. Any fido2luks return code other than 0 (like NoAuthenticatorError when no authenticators are available) will lead to fallback. Fallback to passphrase is enabled by default and may be disabled via FIDO2LUKS_PASSWORD_FALLBACK.

In addition this PR sets a default value for FIDO2LUKS_USE_TOKEN so that the script does not throw errors if it is not configured.

### CODE REVIEW

**Comment by shimunn** [comment]
Sorry for the late response, I'll get to reviewing this tomorrow evening but it looks good so far

File: initramfs-tools/keyscript.sh [hunk 1]
```diff
@@ -11,4 +22,9 @@ if [ "$FIDO2LUKS_USE_TOKEN" -eq 1 ]; then
 	export FIDO2LUKS_CREDENTIAL_ID="$FIDO2LUKS_CREDENTIAL_ID,$(fido2luks token list --csv $CRYPTTAB_SOURCE)"
 fi
 
-fido2luks print-secret --bin
+fido2luks print-secret --bin 2>/dev/null
```
Comment (shimunn) [inline initramfs-tools/keyscript.sh:25]: Hiding potential error messages isn't really helpful please remove `2>/dev/null

**Review by shimunn — CHANGES_REQUESTED** [review]
Looks all good apart from my comment and works as expected :)

File: initramfs-tools/keyscript.sh (hunk 1 above)
Comment (cbachert) [reply initramfs-tools/keyscript.sh]: True, remnant from testing when I was exploring different approaches, including saving stdout/stderr to variables. Saving binary output of fido2luks to shell variables turned out unfeasible and I overlooked that I had left the '2>/dev/null' in the script. Thanks for spotting and merging.

**Pull request merged**